src/bbs2gh/Services/BbsSmbArchiveDownloader.cs

```diff
@@ -205,7 +205,14 @@ private object CreateFileHandle(ISMBFileStore fileStore, string sharedFilePath)
 
         return IsSuccessStatus(status)
             ? sharedFileHandle
-            : throw new OctoshiftCliException($"Couldn't create SMB file handle for \"{sharedFilePath}\" (Status Code: {status}).");
+            : throw new OctoshiftCliException(
+                $"Couldn't create SMB file handle for \"{sharedFilePath}\" (Status Code: {status})." +
+                (IsObjectPathNotFoundStatus(status) && BbsSharedHomeDirectory is BbsSettings.DEFAULT_BBS_SHARED_HOME_DIRECTORY_WINDOWS
```

Comment:
Really nice idea to only append this if the value hasn't been customised!

Reply:
addressed in bf5a5d6dea62301209ea978394089a368aca2ac6